wapitiCore/attack/mod_csp.py

```diff
@@ -65,21 +117,14 @@ async def attack(self, request: Request, response: Optional[Response] = None):
             )
         else:
             csp_dict = csp_header_to_dict(response.headers["Content-Security-Policy"])
-
+            info = ""
             for policy_name in CSP_CHECK_LISTS:
-                result = check_policy_values(policy_name, csp_dict)
-
-                if result <= 0:
-                    if result == -1:
-                        info = MSG_CSP_MISSING.format(policy_name)
-                    else:  # result == 0
-                        info = MSG_CSP_UNSAFE.format(policy_name)
-
-                    log_red(info)
-                    await self.add_vuln_low(
-                        category=NAME,
-                        request=request_to_root,
-                        info=info,
-                        wstg=WSTG_CODE,
-                        response=response
-                    )
+                info += check_policy(policy_name, csp_dict)
+
+            await self.add_vuln_low(
+                category=NAME,
+                request=request,
+                info=info,
+                wstg=WSTG_CODE,
+                response=response
+            )
```

Comment:
Do we always add this vuln ? Even if all the CSP are correctly configured ?

Reply:
Indeed, looks like an indentation issue